packages/base/Form/src/FormField/FormField.tsx

```diff
@@ -47,11 +42,7 @@ const FormFieldAdornments = ({
 
   if (!autoSaveIndicator) {
     return (
-      <div
-        className={cx({
-          [classes.horizontalLayoutAdornment]: layout === 'horizontal',
-        })}
-      >
+      <div className={twJoin(layout === 'horizontal' && '[grid-area:error]')}>
```

Comment:
```suggestion
      <div className={twJoin(isHorizontalLayout && '[grid-area:error]')}>
```

we use the same condition 3 times